### PR TITLE
Add `localizing-datetime` go link

### PR DESCRIPTION
**Pull request by shihaohong**

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -145,7 +145,8 @@
       { "source": "/go/android-plugin-migration", "destination": "https://flutter.dev/docs/development/packages-and-plugins/plugin-api-migration", "type": 301 },
       { "source": "/go/simple-flutter-tool-memory-benchmarks", "destination": "https://docs.google.com/document/d/1vlARGxdABMeZoj4HF0p075-81q7G9gv_GnDYgoF7e3I", "type": 301 },
       { "source": "/go/actions-and-shortcuts-design-revision", "destination": "https://docs.google.com/document/d/1A9yald55O_C6Weqp4tOtyMisfOC1ZC0UMq7Yc-pNfVQ/edit?usp=sharing", "type": 301 },
-      { "source": "/go/scrollable-alert-dialog", "destination": "https://docs.google.com/document/d/1qtVfjoKXE8BtkNbraKk9IzfRcks5btkKcyE4iGVwkbg/edit?usp=sharing", "type": 301 }
+      { "source": "/go/scrollable-alert-dialog", "destination": "https://docs.google.com/document/d/1qtVfjoKXE8BtkNbraKk9IzfRcks5btkKcyE4iGVwkbg/edit?usp=sharing", "type": 301 },
+      { "source": "/go/localizing-datetime", "destination": "https://docs.google.com/document/d/1QR3qLXWdroTVLbnAQ8mq-Ia6tsZ8NY3lmi7gySQAstU/edit?usp=sharing", "type": 301 }
```
Comment (legalcodes) [inline firebase.json]: Dangling `,`

Comment (shihaohong) [reply firebase.json]: Done!

**Comment by sfshaza2** [comment]
@shihaohong, there is a comma on the last line of JSON. You broke our build before because of that.  :). Please remove that last comma and I can merge.

**Comment by shihaohong** [comment]
@Sfshaza Thanks for the heads up! I was wondering about that since I remembered that that had happened before. I think the flutter template even suggests to add the dangling comma, so we'll need to fix that there as well.

File: firebase.json (hunk 1 above)
Comment (shihaohong) [reply firebase.json]: I removed the dangling comma since it caused the build to break before

**Comment by shihaohong** [comment]
Never mind my statement, I'm just bad at English -- didn't realize `penultimate` in [the instructions](https://docs.google.com/document/d/1SFRO8U2toOlAaZ38dsuEU7Wm5fn41wvBCWKiwADqfmw/edit) meant second last, not last :)

**Comment by sfshaza2** [comment]
Yes, the dangling comma must be on every line EXCEPT the last line! Please do fix that in the instructions. Our build was broken for several days last time because you made the change directly to master, so Travis didn't catch it. We've now disallowed making changes directly to master.  :D. Thanks for fixing!

**Comment by sfshaza2** [comment]
Ugh. This is why instructions shouldn't use multisyllabic words!!! Too many people won't understand...

**Pull request merged**